### DISCUSSION TITLE
이미지가 너무 크게 업로드 됩니다

### DISCUSSION THREAD

**Discussion by julia0926** [pronist/hELLO#162] [Q&A] [answered]
안녕하세요 최근 블로그를 다시 작성하려하면서 예쁜 스킨 잘 적용했습니다 ㅎㅎ
다름이아니라 기존 작성글 중 큰 이미지가 있어 그대로 스킨을 적용하니,
그 사진 원본 그대로가 리스트에 보이는데, 혹시 이를 적당한 크기로 작게 보여주시는건 어떠실까요..?
너무 커서 밑의 글들이 밀리는 경향이 있어 작은 저의 의견입니다.
좋은 스킨 만들어주셔서 감사합니다 :)

*제 블로그의 일부분을 캡쳐하였습니다
<img width="273" alt="스크린샷 2022-12-29 오후 10 26 55" src="https://user-images.githubusercontent.com/37897873/209959424-d61e18d5-8da4-4230-a5fe-6f55f0d161c4.png">

**Comment by pronist** [comment, marked as answer]
@julia0926 안녕하세요.

스킨 옵션에서 글 목록 스타일을 적용해보시기 바랍니다.

![1](https://user-images.githubusercontent.com/25624965/209963272-452627dd-11d3-4820-b91b-d7a1c355c9bd.png)

**Reply by julia0926** [reply]
감사합니다~~!
